### PR TITLE
React!!!11

### PR DESCRIPTION
**Pull request by lwille**
TODOs: 
* [x] (feedback required): map vowel position names to the actual values, so we can use fancy `<select>`s. Should probably be done in `src/Vowel.js` (which could perhaps contain a map like `{"frontness":"Near-Front": 0.33}` @jleeothon 
* [x] review app is not building properly, I'm getting sleepy so I'll fix it mañana. Works nice locally though, give it a try!

Squashed away the [Four Noble Truths](https://en.wikipedia.org/wiki/Four_Noble_Truths) commit messages.

Prerequisite for #10

### CODE REVIEW

**Comment by lwille** [comment]
update: haven't seen https://github.com/schweinebauch-heroes/vowelchart/compare/positions when writing this MR 💯

**Comment by jleeothon** [comment]
The "positions" branch is not polished 🤔 . Let's give it some proper thought.

**Comment by jleeothon** [comment]
The polished list of vowels and positions was more polished and merged

File: .eslintrc.js [hunk 1]
```diff
@@ -8,10 +8,14 @@ module.exports = {
     "plugin:react/recommended",
     "plugin:jest/recommended",
   ],
+  'globals': {
+    'it': false,
```
Comment (jleeothon) [inline .eslintrc.js]: I think this isn't needed anymore?

File: src/App.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ import './App.css'
 import Editor from './views/Editor'
 import VowelPlotter from './VowelPlotter'
 
+import {list} from './VowelList'
```
Comment (jleeothon) [inline src/App.js]: The exports here changed a bit 🙈 

You can use `import vowels from './VowelList'` now, or whatever name you'd like for the default export.

**Comment by lwille** [comment]
Thanks for review so far; I've continued implementing dynamic rendering of editor components - now the value names from `Articulation.js` are being used and we directly attach the numerical values to the `<option>` elements - problem solved!

**Comment by lwille** [comment]
Changing buildpack to https://github.com/mars/create-react-app-buildpack.git helped.
We shouldn't forget to add a node.js buildpack again if we want to run a backend server (however that can also be deployed as separate heroku app).

**Pull request merged**